great_expectations/core/batch.py

```diff
@@ -872,50 +873,42 @@ def get_batch_request_from_acceptable_arguments(
 def standardize_batch_request_display_ordering(
     batch_request: Dict[str, Union[str, int, Dict[str, Any]]]
 ) -> Dict[str, Union[str, Dict[str, Any]]]:
-    datasource_name: str = batch_request["datasource_name"]
-    data_connector_name: str = batch_request["data_connector_name"]
-    data_asset_name: str = batch_request["data_asset_name"]
-    runtime_parameters: str = batch_request.get("runtime_parameters")
-    batch_identifiers: str = batch_request.get("batch_identifiers")
-    batch_request.pop("datasource_name")
-    batch_request.pop("data_connector_name")
-    batch_request.pop("data_asset_name")
-    # NOTE: AJB 20211217 The below conditionals should be refactored
+    datasource_name: Optional[str] = batch_request.get("datasource_name")
+    data_connector_name: Optional[str] = batch_request.get("data_connector_name")
+    data_asset_name: Optional[str] = batch_request.get("data_asset_name")
+    runtime_parameters: Optional[str] = batch_request.get("runtime_parameters")
+    batch_identifiers: Optional[str] = batch_request.get("batch_identifiers")
+    if datasource_name is not None:
```

Comment:
@NathanFarmer Stylistic (minor) -- could we please put the datasource processing first (the ordering of the triumvirate for easier reading).  Thanks!

Reply:
@alexsherstinsky I'm confused. Am I not processing everything in the correct order here? 1. `datasource_name`, 2. `data_connector_name`, 3. `data_asset_name`